xmake/platforms/wasm/xmake.lua

```diff
@@ -40,3 +40,20 @@ platform("wasm")
     -- set toolchains
     set_toolchains("emcc")
 
+    -- set menu
+    set_menu {
+        config =
+        {
+            {category = "Emscripten Configuration"                    }
+        ,   {nil, "emsdk", "kv", nil, "emsdk directory" }
+        ,   {nil, "emscripten", "kv", nil, "emscripten directory" }
```

Comment:
we only need emsdk argument.

Reply:
from my understanding, you can have the emscripten toolchain without emsdk, that's why I separated them.
emsdk may be useful to install/update the emscripten toolchain where only the emscripten toolchain is required to build

Reply:
> from my understanding, you can have the emscripten toolchain without emsdk, that's why I separated them.

But I have never seen a stand-alone emscripten toolchain.

I only know about installing and using emscripten in emsdk. Are there other installers and tarballs that only include emscripten without emsdk? I don't seem to have seen any.

Reply:
I didn't see any either but they seem to be mostly isolated, emscripten folder doesn't seem to depend on emsdk folder. One can download emscripten without using emsdk which could be useful to add it as a toolchain.
Separating them seemed like the safest and most flexible option.

Reply:
This is similar to the relationship between vs/msvc, qtsdk/clang, but only emscripten is included in emsdk, and only emsdk is officially provided to install emscripten.

I don't think any other separate emscripten sdk will be distributed.

We could start by just find_emsdk to maintain the paths to emsdk and emscripten, which would be easier to maintain. Even if we come across a separate emscripten sdk later, it would be fine to go back and add it.

Also, we now have an emcc toolchain。

Reply:
As you wish, I will update it